### PR TITLE
feat: mock async req/res in acceptance tests

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: test/acceptance/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+BrowserStack is used to run the tests to reduce complexity and provide an
+improved experience that would otherwise be possible running tests locally.
+
+## Setup
+
+Need to have environment variables made available to the container. Created `.env` file in acceptance test directory.
+
+Env vars:
+
+TODO: add table
```
Comment (johnwatson484) [inline test/acceptance/README.md:24]: Is this not already done below?

**Review by johnwatson484 — COMMENTED** [review]
Quick query on the README.

File: test/acceptance/README.md (hunk 1 above)
Comment (st3v3nhunt) [reply test/acceptance/README.md:24]: It is. Let me get that updated.

Comment (st3v3nhunt) [reply test/acceptance/README.md]: Good spot. I'd meant to remove that section as it was to remind me to add it later in the file.
It has now been removed.

**Review by johnwatson484 — APPROVED** [review]
Approved

**Pull request merged**